python/examples/tabulated.py

```diff
@@ -0,0 +1,156 @@
+"""
+Tabulated radial integrals
+==========================
+
+.. start-body
+"""
+
+# This script illustrates how to generate splined radial basis functions/integrals,
+# using a "rectangular" LE basis as the example, i.e, a LE basis truncated with
+# l_max, n_max hyperparameters.
+
+
+import ase
+import numpy as np
+import scipy as sp
+from numpy import arange, pi, sqrt, zeros
+from scipy.optimize import brentq
+from scipy.special import jv
+from scipy.special import spherical_jn as j_l
+
+from rascaline import SphericalExpansion, generate_splines
+
+
+########################################################
+# Spherical Bessel zeros (from the scipy cookbook)
+
+
+def Jn(r, n):
+    return sqrt(pi / (2 * r)) * jv(n + 0.5, r)
+
+
+def Jn_zeros(n, nt):
+    zerosj = zeros((n + 1, nt), dtype=np.float64)
+    zerosj[0] = arange(1, nt + 1) * pi
+    points = arange(1, nt + n + 1) * pi
+    racines = zeros(nt + n, dtype=np.float64)
+    for i in range(1, n + 1):
+        for j in range(nt + n - i):
+            foo = brentq(Jn, points[j], points[j + 1], (i,))
+            racines[j] = foo
+        points = racines
+        zerosj[i][:nt] = racines[:nt]
+    return zerosj
+
+
+########################################################
+
+
+# Hyperparameters:
+lmax = 10
+nmax = 10
+a = 5.0  # Radius of the LE sphere - i.e, the cutoff radius
```

Comment:
I would keep the notation consistent with everything else:

```suggestion
max_angular = 10
max_radial = 10
cutoff = 5.0  # This is also the radius of the LE sphere
```